pact.cabal

```diff
@@ -179,6 +183,7 @@ library
     , prettyprinter-ansi-terminal >= 1.1 && < 1.2
     , quickcheck-instances >= 0.3
     , reflection
+    , scientific >= 0.3
```

Comment:
It will allow to unpin hashable and related dependencies, such as aeson-2, assuming that backward compatibility issues of related dependencies are fixed.

Unfortunately, upgrading hashable also requires updating `prettyerinter` and `trifecta`, which in turn requires to implement a solution for backward compatible pact user error formatting. I've made a proposal to deal with that [here](https://github.com/kadena-io/chainweb-issues/issues/30).

So, there's more work to do until we are caught up with recent Hackage versions.